test/tests/concrete_expansion_microcracking/tests

```diff
@@ -0,0 +1,88 @@
+    csvdiff = 'concrete_expansion_microcracking_free_out.csv'
+    abs_zero = 1e-6
+    cli_args = 'Outputs/file_base=concrete_expansion_microcracking_free_out'
+    allow_test_objects = True
+  [../]
+
+  [./concrete_free_shrinkage]
+    type = 'CSVDiff'
+    input = 'concrete_expansion_microcracking.i'
+    csvdiff = 'concrete_expansion_microcracking_shrinkage_out.csv'
+    abs_zero = 1e-6
+    cli_args = 'Functions/expansion_vs_time/value="-max(0, t - 0.0001)"
+                Materials/microcracking/eigenstrain_factor=-1.0
+                Outputs/file_base=concrete_expansion_microcracking_shrinkage_out'
+    allow_test_objects = True
+  [../]
+
+  [./concrete_expansion_load_5]
+    type = 'CSVDiff'
+    input = 'concrete_expansion_microcracking.i'
+    csvdiff = 'concrete_expansion_microcracking_5_out.csv'
+    abs_zero = 1e-6
+    cli_args = 'BCs/load_x/value=-5e6
+                Outputs/file_base=concrete_expansion_microcracking_5_out'
+    allow_test_objects = True
+  [../]
+
+  [./concrete_expansion_load_10]
+    type = 'CSVDiff'
+    input = 'concrete_expansion_microcracking.i'
+    csvdiff = 'concrete_expansion_microcracking_10_out.csv'
+    abs_zero = 1e-6
+    cli_args = 'BCs/load_x/value=-10e6
+                Outputs/file_base=concrete_expansion_microcracking_10_out'
+    allow_test_objects = True
+  [../]
+
+  [./concrete_expansion_load_5_5]
+    type = 'CSVDiff'
+    input = 'concrete_expansion_microcracking.i'
+    csvdiff = 'concrete_expansion_microcracking_5_5_out.csv'
+    abs_zero = 1e-6
+    cli_args = 'BCs/load_x/value=-5e6
+                BCs/load_y/value=-5e6
+                Outputs/file_base=concrete_expansion_microcracking_5_5_out'
+    allow_test_objects = True
+  [../]
+
+  [./concrete_expansion_load_10_10]
+    type = 'CSVDiff'
+    input = 'concrete_expansion_microcracking.i'
+    csvdiff = 'concrete_expansion_microcracking_10_10_out.csv'
+    abs_zero = 1e-6
+    cli_args = 'BCs/load_x/value=-10e6
+                BCs/load_y/value=-10e6
+                Outputs/file_base=concrete_expansion_microcracking_10_10_out'
+    allow_test_objects = True
+  [../]
+
+  [./concrete_expansion_load_5_5_5]
+    type = 'CSVDiff'
+    input = 'concrete_expansion_microcracking.i'
+    csvdiff = 'concrete_expansion_microcracking_5_5_5_out.csv'
+    abs_zero = 1e-6
+    cli_args = 'BCs/load_x/value=-5e6
+                BCs/load_y/value=-5e6
+                BCs/load_z/value=-5e6
+                Outputs/file_base=concrete_expansion_microcracking_5_5_5_out'
+    allow_test_objects = True
+  [../]
+
+  [./concrete_expansion_load_10_10_10]
+    type = 'CSVDiff'
+    input = 'concrete_expansion_microcracking.i'
+    csvdiff = 'concrete_expansion_microcracking_10_10_10_out.csv'
+    abs_zero = 1e-6
+    cli_args = 'BCs/load_x/value=-10e6
+                BCs/load_y/value=-10e6
+                BCs/load_z/value=-10e6
+                Outputs/file_base=concrete_expansion_microcracking_10_10_10_out'
+    allow_test_objects = True
```

Comment:
We have a new system in MOOSE for our software quality that requires that every test provide 3 parameters: a requirement, and issue, and a reference to a design document (the .md file). They're requiring that in all new tests in MOOSE, and we'll start doing that in BlackBear at some point. I'm going to go ahead and merge this, but it will be easiest to get this added to these tests (and others that you have added) while it's fresh on your mind.  I'd really appreciate it if you could do that.  Look in moose/modules/xfem for examples (just because we recently added that stuff to every test in there).